src/modules/print.js

```diff
@@ -115,54 +113,47 @@ export const showTasks = () => {
     listItemTitle.innerHTML = `${task.description}`;
     tasksDiv.appendChild(listItemTitle);
 
-    // Create a button to remove the book
     const removeButton = document.createElement('span');
     removeButton.setAttribute('class', 'close');
-    removeButton.getAttribute('id', 'remove-icon');
+    removeButton.setAttribute('id', 'remove-icon');
     removeButton.innerHTML = '🗑';
```

Comment:
- [ ] kindly consider updating the local storage when a user deletes a todo with this icon so as to meet the below requirement
> The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads. Make sure to reload the page to test the array is stored.

consider doing the same even on the check boxes status by making sure that if the user updates the list when the UI is changing it also updates the local storage's  completed status

Reply:
Save task is added to the code section to update local storage.
[
![image](https://github.com/Kaiserabbas/todo-list/assets/123939543/cc4d65da-a55c-421e-95a3-f0d98fd6cae8)
](url)